src/app/cube/cube.routing.ts

```diff
@@ -65,6 +66,7 @@ const cube_routes: Routes = [
         data: { breadcrumb: 'Preferences' },
         canActivate: [AuthGuard]
       },
+      { path: 'details/error-status', component: ErrorStatusComponent },
```

Comment:
@EVogel1999 what do you think about just having a template conditional to show this component? That way it doesn't route to a new page.

I believe this implementation was taken from the current 404 page, but it seems to be kind of confusing for users.